Apps/phironnet/ironnet.json

```diff
@@ -510,6 +636,1335 @@
                 "type": "table"
             },
             "versions": "EQ(*)"
+        },
+        {
+            "action": "get irondome info",
```

Comment:
Please change to `get info`

Reply:
`get info` alone would be insufficient, as there are other endpoints that supply different types of information.

Reply:
Can it be changed to `get alert info`? To create more general action names, we avoid adding product and/or vendor names

Reply:
Would `get_community_info` be agreeable?

Reply:
Yes, that works!

Reply:
Perfect - done.